airbyte-integrations/connectors/source-stripe/source_stripe/spec.json

```diff
@@ -9,7 +9,6 @@
     "properties": {
       "client_secret": {
         "type": "string",
-        "pattern": "^(s|r)k_(live|test)_[a-zA-Z0-9]+$",
```

Comment:
what's the reason to remove this? it's better to give feedback instantly if we know it's correct

Reply:
I've found that I'm unable to delete Stripe source in Airbyte UI (and I'm [not alone](https://airbytehq.slack.com/archives/C01MFR03D5W/p1634587315164500)). Also, I found this redundant because there is a clear example in the settings page of the connector for this key and Stripe can change key-generating logic at any moment. 

Seems that the real reason for that bug is somewhere in backend logic which isn't related to stripe connector, so it's easier to delete excessive validation. Btw you can reproduce it — just add the latest stripe source, configure It, try to delete and open web inspector in your browser to see errors.

Reply:
thanks for the context. this seems like a fair workaround then. Ive created an issue to address the root cause: https://github.com/airbytehq/airbyte/issues/8244